ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -958,24 +961,39 @@ function ListRules(props) {
 
   useEffect(() => {
     if (drawerReleaseName) {
-      Promise.all([
-        fetchRelease(drawerReleaseName),
-        fetchScheduledChangeByName(drawerReleaseName),
-      ]).then(([fetchedRelease, fetchedSC]) => {
-        const item =
-          fetchedSC.data.data.count > 0
-            ? JSON.stringify(
-                fetchedSC.data.data.scheduled_changes[0].data,
-                null,
-                2
-              )
-            : JSON.stringify(fetchedRelease.data.data, null, 2);
-
-        setDrawerState({
-          ...drawerState,
-          open: true,
-          item,
-        });
+      fetchReleaseV2(drawerReleaseName).then(fetchedRelease => {
+        if (!fetchedRelease.error) {
+          const item =
+            Object.keys(fetchedRelease.data.data.sc_blob).length > 0
+              ? JSON.stringify(fetchedRelease.data.data.sc_blob, null, 2)
+              : JSON.stringify(fetchedRelease.data.data.blob, null, 2);
+
+          setDrawerState({
+            ...drawerState,
+            open: true,
+            item,
+          });
+        } else {
+          Promise.all([
+            fetchRelease(drawerReleaseName),
+            fetchScheduledChangeByName(drawerReleaseName),
+          ]).then(([fetchedRelease, fetchedSC]) => {
+            const item =
+              fetchedSC.data.data.count > 0
+                ? JSON.stringify(
+                    fetchedSC.data.data.scheduled_changes[0].data,
+                    null,
+                    2
+                  )
+                : JSON.stringify(fetchedRelease.data.data, null, 2);
+
+            setDrawerState({
+              ...drawerState,
+              open: true,
+              item,
+            });
+          });
+        }
       });
     }
   }, [drawerReleaseName]);
```

Comment:
STR:

1. Navigate to https://localhost:9000/rules while being logged out
2. Click the "Disable Updates" speed dial button
3. Open the developer tools and see error: `TypeError: productChannelQueries is null`

Reply:
Hm, we shouldn't be showing this button if you're logged out. I'm not sure if that's a new issue or not, but it's easy enough to fix here.

Reply:
Sounds good. It's possible that it was from before. Disabling it when logged out makes sense to me.

Reply:
I wasn't able to reproduce this myself, but there's clearly an unguarded access of `productChannelQueries` in this file, which I'm fixing.

Reply:
Looks like this was actually caused by an invalid use of `SpeedDialAction` -- `disabled` needs to be in `FabProps`, not its own attribute. Looks like this one was missed during #1150.